CVE-2014-0160/cvex.yml

```diff
@@ -0,0 +1,8 @@
+blueprint: debian7-debian7
```

Comment:
Do you need Debian only for ifupdown to configure network interfaces? Where is that in the code? Why not to use Ubuntu's /etc/netplan? If you **really** need Debian 7, please also provide a pull request for a new blueprint in https://github.com/ucsb-seclab/CVEX.

Reply:
I will submit a corresponding pull request to the other repo. Debian doesn't support netplan since it is Ubuntu specific.